### ISSUE TITLE
0.7.5.1 Crash on first time joining server

### ISSUE THREAD

**Issue by Lussian-A** [StrangeLoopGames/EcoIssues#8419]
I keep gettin this crash every time I join a for the first time; 

---------
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	Eco.exe
  Application Version:	2017.4.2.55755
  Application Timestamp:	5ae30bb2
  Fault Module Name:	ntdll.dll
  Fault Module Version:	6.1.7601.24094
  Fault Module Timestamp:	5abee643
  Exception Code:	c0000005
  Exception Offset:	000000000006b766
  OS Version:	6.1.7601.2.1.0.256.1
  Locale ID:	2057
  Additional Information 1:	f7a2
  Additional Information 2:	f7a2d2b5abe548332644309259aebf87
  Additional Information 3:	ee67
  Additional Information 4:	ee67a736e7942bcd4bf4d41d7a5a72db
----------

Best deductive guess would be it has something to do with the character creation/customization screen. Since if I like in a 2nd time, I can enter without a crash.. but by then you don't get the first time character screen <.<

**Comment by BlazeGuy2** [comment]
Could you try to trigger this crash again and send us the log file?  Also does it crash as well when you try to access the character customization screen again in game (open backpack and click the green edit button underneath the characters clothing and 3rd person toggle)?

**Comment by Hydrohawk** [comment]
Myself and several others have encountered this error as well, always on first time logging into a new server.  The game will try to load up normally, and then crash to desktop.  For people in-game, it will appears that someone has logged on for the first time, then they promptly log off.  This results in skipping the entire character customization screen.

Not sure where to find a log file for the crash.

**Comment by johnkslg** [comment]
@Hydrohawk @Lussian-A Please message me on Discord.gg/eco @johnk
Working on this now

**Comment by MarjieAVolk** [comment]
If either of you could send more logs and/or crash files from these crashes, that would be really helpful.  These are located in C:\Users\\[username]\AppData\LocalLow\Strange Loop Games\ECO

**Comment by Hydrohawk** [comment]
Found the folder.  What file do you need.  I don't see a crash record for yesterday when I had this happen to me.

**Comment by MarjieAVolk** [comment]
Do you have a Crashes folder in there?  Anything from around the time you saw the issue?

If you do see the issue again, then the output_log.txt file will be helpful, but that gets overwritten every play session.

**Comment by Hydrohawk** [comment]
[output_log.txt](https://github.com/StrangeLoopGames/EcoIssues/files/2181702/output_log.txt)

**Comment by Hydrohawk** [comment]
There was no crash, but I never exited the game when I popped on a new server last night.

**Comment by Hydrohawk** [comment]
The last crash was dated the 2nd of July.  I'll upload those as well.

**Comment by Hydrohawk** [comment]
[Crash_2018-07-02_212724.zip](https://github.com/StrangeLoopGames/EcoIssues/files/2181705/Crash_2018-07-02_212724.zip)
[Crash_2018-07-02_233800.zip](https://github.com/StrangeLoopGames/EcoIssues/files/2181708/Crash_2018-07-02_233800.zip)

**Comment by MarjieAVolk** [comment]
Hm, your log has a whole lot of these, that's interesting:
```
NullReferenceException: Object reference not set to an instance of an object
  at Player.get_PosEco () [0x00005] in <cd6309eb490d44a3ba0372cec42e98f7>:0 
  at AdjustTransformToCurve.LateUpdate () [0x00060] in <cd6309eb490d44a3ba0372cec42e98f7>:0 
```

Probably not related to the crash, since you were playing after the crash?  But looks kinda bad.

**Comment by Lussian-A** [comment]
Here's one from me when it happens.

[output_log.txt](https://github.com/StrangeLoopGames/EcoIssues/files/2181762/output_log.txt)

**Comment by MarjieAVolk** [comment]
Thanks.  Okay yeah I don't see anything bad in that log file, so I'll assume the errors from @Hydrohawk 's are unrelated.

Do you have any crash files?

**Comment by Lussian-A** [comment]
About the only one around that time. Rest are old unrelated. 

[Crash_2018-07-02_165216.zip](https://github.com/StrangeLoopGames/EcoIssues/files/2181817/Crash_2018-07-02_165216.zip)

**Comment by SeanAtr0n** [comment]
The game crashed both of those times? Nothing went into the crashes folder?

**Comment by Lussian-A** [comment]
Just tested 0.7.6.0 staging version, joined the one server in existence. That works without a hitch.
Going to try to redownload 0.7.5.1 just to see if it was a bad download or something.

**Comment by Hydrohawk** [comment]
@SeanAtr0n The game crashed on the first attempt to join a server.  On the second attempt, there were no issues...other than what appeared to be a lot of lag (very built-up world).

**Comment by SeanAtr0n** [comment]
Fixed in 7.6, and i have a fix for 7.5.1 as well but it sounds like we won't be shipping that
